### DISCUSSION TITLE
Multiple clients in Next.js not supported?

### DISCUSSION THREAD

**Discussion by feelixe** [trpc/trpc#4126] [Q&A]
Issue #2121 says that you can use multiple clients. Is this not supported in Next.js yet?
Because when I do:
_app.tsx
```typescript
const standard = trpc.withTRPC(MyApp);
const background = trpcBackground.withTRPC(standard);
export default background;
```
And then
```typescript
const foo1 = trpc.foo.useMutation();
const foo2 = trpcBackground.foo.useMutation();
```
Both `foo1` and `foo2` will send a request to the URL that `trpc` uses and not `trpcBackground`.

**Comment by KATT** [comment]
If you do, you likely might need to setup the providers manually as done in the test in #3049

**Reply by feelixe** [reply]
Would it be relevant to open an issue for supporting multiple clients in `@trpc/next`'s `trpc.withTRPC`?
Feels like quite a hassle to manually add everything that `.withTRPC` wraps the app with and make sure it stays up to date with changes to the tRPC package.
